codespell.txt

```diff
@@ -3,3 +3,4 @@ heist
 Hist
 Heist
 his
+ro
```

Comment:
So codespell flagged the `ro` (return object varibale) in the new coroutine flowchart. I could not figure out how to silence codespell on just one line, so I added ro to the exceptions here. Does anyone know a better way?

Reply:
In my experience, automatic codespell is bound to fail. So we can keep it as an hint, but we'll have to ignore it most of the time. Never seen a better way

Reply:
What do you mean with "keep it as an hint"? Should I add a comment in the codespell.txt why I added `ro` to the exceptions?

Reply:
I mean keep the codespell so that we can go through its output from time to time and fix what needs to be fixed. But ignore things like "ro". By "hint", I meant something that does not constrain us but helps finding out problems.

Anyway, for now, let's merge this

Reply:
Hmm. But having a CI job that we are ignoring anyway, because it's always red, is also not a good practice :) I would prefer if we can keep it green.

Reply:
everybody would, but you've just demonstrated it will be tough. And this is my experience indeed with spell checking